scripts/nbd_client_manager.py

```diff
@@ -114,8 +114,9 @@ def _find_unused_nbd_device():
     """
     Returns the path of the first /dev/nbdX device that is not
     connected according to nbd-client.
+    Raises NbdDeviceNotFound if no devices are available.
     """
-    for device_no in range(0, 100):
+    for device_no in range(0, 1000):
```

Comment:
you use 1000 here and 128 below, should this be 128 too?

Reply:
No, this is deliberate, to keep the old behaviour: when we go past 128, we'll get an exception due to nonexistent nbd device that we catch in the loop. This is to just keep the code simple.